samples/nextjs/src/pages/[[...path]].tsx

```diff
@@ -18,7 +19,7 @@ const componentPropsService = new ComponentPropsService();
 const SitecorePage = ({ layoutData, componentProps }: SitecorePageProps): JSX.Element => {
   if (!layoutData?.sitecore?.route) {
     // layoutData will be missing for an invalid path
-    return <Error statusCode={404} />;
+    return <NotFound context={layoutData?.sitecore.context} />;
```

Comment:
Could sitecore be null/undefined?
```suggestion
    return <NotFound context={layoutData?.sitecore?.context} />;
```

Reply:
@ambrauer It can't be null/undefined, because you check `if (!layoutData?.sitecore?.route) {` above (line 20)